en/docs/insights/choreo-insights-api.md

```diff
@@ -1162,6 +1162,207 @@ Time filter
 
 Successful API usage by application filter.
 
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getSuccessAPIsUsageByApplicationsWithOnPremKey</strong></td>
+<td valign="top">[<a href="#successapiusagebyapplication">SuccessAPIUsageByApplication</a>]</td>
+<td>
+
+Returns a list of successful hit counts (i.e., 2xx response codes) within the specified time range for each API,
+grouped by each application. Optionally, you can configure the <code>successAPIUsageByAppFilter</code> parameter to further
+filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">onPremKey</td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Value of the On-Prem key
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
```

Comment:
Some descriptions have one or more sentences with punctuation while some descriptions like this have just two words and no punctuation.

To be consistent, convert sentences like this to sentences by adding an article in front and punctuation.

e.g., this description can be changed to "The time filter.".

This comment applies to all the similar descriptions in this document.